### PR TITLE
openjdk@8 1.8.0+282

### PR DESCRIPTION
**Pull request by bayandin**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

**Comment by fxcoudert** [comment]
`voldemort` failure:

```
==> /usr/local/Cellar/voldemort/1.10.26_1/bin/vadmin.sh
The operation couldn’t be completed. Unable to locate a Java Runtime.
```

Given it has 2 monthly installs and was last updated in 2017, we could disable it

**Comment by bayandin** [comment]
> `voldemort` failure:
> 
> ```
> ==> /usr/local/Cellar/voldemort/1.10.26_1/bin/vadmin.sh
> The operation couldn’t be completed. Unable to locate a Java Runtime.
> ```
> 
> Given it has 2 monthly installs and was last updated in 2017, we could disable it

I think setting JAVA_HOME should help, let me try to do it

File: Formula/voldemort.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Voldemort < Formula
   url "https://github.com/voldemort/voldemort/archive/release-1.10.26-cutoff.tar.gz"
   sha256 "8bd41b53c3b903615d281e7277d5a9225075c3d00ea56c6e44d73f6327c73d55"
   license "Apache-2.0"
-  revision 1
+  revision 2
```
Comment (carlocab) [inline Formula/voldemort.rb:7]: Is the revision bump needed, or is setting `JAVA_HOME` in the test enough?

Comment (SMillerDev) [reply Formula/voldemort.rb:7]: It's in the install so this is needed

**Review by SMillerDev — APPROVED** [review]

File: Formula/voldemort.rb (hunk 1 above)
Comment (carlocab) [reply Formula/voldemort.rb:7]: Ah, misread. Thought it was the test block

**Pull request closed without merging**